app/assets/stylesheets/modules/_landing.scss

```diff
@@ -0,0 +1,41 @@
+.landing {
+  .banner {
```

Comment:
Ah OK I see your point about name spacing :) I think `.landing-banner` would be clearer since we can immediately tell which namespace it is under.